packages/marko-web-identity-x/browser/profile.vue

```diff
@@ -176,6 +176,20 @@
         An error occurred: {{ error }}
       </p>
     </form>
+    <div v-else>
+      <div class="success-message">
+        <div class="success-message__title">
+          Your profile has been saved.
+        </div>
+        <div v-if="returnTo" class="success-message__message">
+          If you are not automatically redirected <a :href="returnTo">click here</a> to continue.
+        </div>
+        <div v-else class="success-message__message">
+          <a href="#" @click="handleReload()">Click here</a> to continue modiying your profile
+          or <a href="/">click here</a> to return to the home page.
```

Comment:
Make this a button with `btn-link` class rather than an A tag with a search param. `modiying` should be `modifying`, and I'd recommend this text:
> [Click here]() to make further changes to your preferences, or [return to the home page]().